docs/source/releases/v3.2.rst

```diff
@@ -39,24 +43,61 @@ Minor changes
 
 - Added a new helper ``core.utils.is_ajax`` which replicates the logic of Django's ``HttpRequest.is_ajax``
   method that was deprecated in Django 3.1.
+  
 - Persisted ``OrderFactory.date_placed`` to the database when set, instead of just setting it on the object without saving it.
 
+- Fixed getting non public categories with ``get_categories`` by only returning browsable categories.
+
+- Reviews can now only be viewed and placed when the product is public. Before you could access the review urls if you knew them or they were indexed into a search engine.
+
+?- Remove title from shippingaddress
+?- Remove title from billing address
+
+- Change ``PartnerProductFilterMixin`` queryset to also include child products for the partner user.
+
+- Let the ``RangeDetailView`` listen to the ``OSCAR_PRODUCTS_PER_PAGE`` setting for the amount of products to be shown on the range page.
+
+
 .. _dependency_changes_in_3.2:
 
 Dependency changes
 ~~~~~~~~~~~~~~~~~~
 
 Python package dependencies:
 
+- Upgraded ``django-environ`` to version 0.9.
+- Upgraded ``django-redis`` to version 5.3.
+- Upgraded ``redis`` to version 4.2.
+- Upgraded ``whitenoise`` to version 5.4.
+- Upgraded ``flake8`` to version 4.0.1.
+- Upgraded ``isort`` to version 5.10.1.
+- Upgraded ``ipython`` to version 9.
 
 Javascript and CSS dependencies:
 
+- Upgraded ``@fortawesome/fontawesome-free`` to version 5.15.4.
+- Upgraded ``bootstrap`` to version 4.6.1.
+- Upgraded ``eslint`` to version 8.8.0.
```

Comment:
We don't need to list internal (development) dependencies here - so you can remove eslint, gulp-sass, node-sass, as these are only used for compilation of assets. We just need to list the libraries that Oscar ships with. (Again, this is to try and keep the release notes as concise as possible so it is easy for people to find the bits that matter to them).